src/ol/interaction/selectinteraction.js

```diff
@@ -13,6 +14,59 @@ goog.require('ol.interaction.Interaction');
 goog.require('ol.style.Style');
 
 
+/**
+ * @enum {string}
+ */
+ol.SelectEventType = {
+  /**
+   * Triggered when feature(s) has been (de)selected.
+   * @event ol.SelectEvent#select
+   * @api
+   */
+  SELECT: 'select'
+};
+
+
+
+/**
+ * @classdesc
+ * Events emitted by {@link ol.interaction.Select} instances are instances of
+ * this type.
+ *
+ * @param {string} type The event type.
+ * @param {ol.Collection.<ol.Feature>} features Currently selected features
+ * @param {Array.<ol.Feature>} selected Selected features
+ * @param {Array.<ol.Feature>} deselected Deselected features
+ * @extends {goog.events.Event}
+ * @constructor
+ */
+ol.SelectEvent = function(type, features, selected, deselected) {
+  goog.base(this, type);
+
+  /**
+   * Currently selected features
+   * @type {ol.Collection.<ol.Feature>}
+   * @api
+   */
+  this.features = features;
```

Comment:
I think this is redundant, since you can also use `ol.interaction.Select#getFeatures()`.